.github/workflows/test-slsa-release.yml

```diff
@@ -65,12 +58,12 @@ jobs:
       # Download build from previous job
       - uses: 'actions/download-artifact@v2'
         with:
-          name: main
+          name: SLSA_BINARY
```

Comment:
How about using outputs of the action `steps.build.outputs.artifact_name`

Reply:
will do, thanks!